Woocommerce download readfile chunk number hook proposal

khanrn opened the pull request:
I think a filter hook (my proposed name is `woocommerce_download_readfile_chunk_number`) should be added to make modifying chunk number easily.

khanrn committed:
Merge pull request #1 from woocommerce/master

Routine Update

khanrn committed:
Merge pull request #2 from woocommerce/master

Routine Update

khanrn committed:
Merge pull request #3 from woocommerce/master

Routine Update

khanrn committed:
Merge pull request #4 from woocommerce/master

Routine Update

khanrn committed:
Merge pull request #5 from woocommerce/master

Routine Update

khanrn committed:
Merge pull request #6 from woocommerce/master

Routine Update

khanrn committed:
Merge pull request #7 from woocommerce/master

Routine Update

khanrn committed:
A hook proposal to modify the chunk number.

`woocommerce_download_readfile_chunk_number` hook is proposed to make modifying the chunk number easily.

mikejolley commented:
Better to use a constant IMO since it's very dev/server oriented. WC_CHUNK_SIZE? No need to set in our code, just see if it's defined and use if set.

khanrn committed:
Constant IMO declared for changeable chunk size. 

New constant `WC_CHUNK_SIZE` declared to make `WC_Download_Handler` class's `readfile_chunked` method's chunk size changeable.

rnaby commented:
That's a good idea. I did the changes. Hopefully now it's OK. Please check. @mikejolley
